assets/css/editor.scss

```diff
@@ -60,3 +60,17 @@
 	margin: 0;
 	padding: 0;
 }
+
+.theme-twentytwenty {
+	.wp-block {
+		h2.wc-block-grid__product-title {
```

Comment:
You will need to remove the `h2` from here. Otherwise, when editing the All Products block and changing the Product Title tag from `h2` to `h3`, the title becomes huge again.

Reply:
Good catch! I removed the `h2` from the CSS selector to affect other heading levels as well.